server/app-routers/admin-links.js

```diff
@@ -3,4 +3,48 @@ const db = require('../database')
 
 const router = express.Router()
 
+router.get('/all', function (req, res) {
+  db.link.findAll({attributes: ['college','uuid','scheduledTime','notesFromCollege','notesFromCollegeSeen','lastSignedIn']}).then(links => {
+      res.json({success: true, data: links})
+  }).catch(function (err){
+    res.json({success: false, message: err.message})
+  })
+})
+
+router.post('/', function (req, res) {
+  db.link.create({
+  	college: req.body.collegeName,
+  	repName: req.body.repName,
+  	tierPriority: req.body.tierPriority,
+  	notesToCollege: req.body.toCollege
+  }).then(linkObject => {
+    res.json({success: true, uuid: linkObject.uuid})
+  }).catch(function (err){
+    res.json({success: false, message: err.message})
+  })
+})
+
+router.delete('/:linkid', function (req, res) {
```

Comment:
This needs to follow the middleware that handles `/upcoming` or otherwise this will catch that too